src/model/documentselection.js

```diff
@@ -723,15 +723,15 @@ function clearAttributesStoredInElement( changes, batch, document ) {
 
 	// `changes.range` is not set in case of rename, root and marker operations.
 	// None of them may lead to the element becoming non-empty.
-	if ( !changeParent || changeParent.isEmpty ) {
+	if ( !changeParent || changeParent.is( 'documentFragment' ) || changeParent.isEmpty ) {
```

Comment:
Is it still needed? I see now change is fired only for docment changes.

Reply:
I think you are right.